moto/databrew/models.py

```diff
@@ -49,6 +71,52 @@ def get_recipe(self, recipe_name):
             raise RecipeNotFoundException(recipe_name)
         return self.recipes[recipe_name]
 
+    def create_ruleset(
+        self, ruleset_name, ruleset_description, ruleset_rules, ruleset_target_arn, tags
+    ):
+        if ruleset_name in self.rulesets:
+            raise RulesetAlreadyExistsException()
+
+        ruleset = FakeRuleset(
+            self.region_name,
+            ruleset_name,
+            ruleset_description,
+            ruleset_rules,
+            ruleset_target_arn,
+            tags,
+        )
+        self.rulesets[ruleset_name] = ruleset
+        return ruleset
+
+    def update_ruleset(self, ruleset_name, ruleset_description, ruleset_rules, tags):
+        if ruleset_name not in self.rulesets:
+            raise RulesetNotFoundException(ruleset_name)
+
+        ruleset = self.rulesets[ruleset_name]
+        if ruleset_description:
+            ruleset.description = ruleset_description
+        if ruleset_rules:
+            ruleset.rules = ruleset_rules
+        if tags:
+            ruleset.tags = tags
+
+        return ruleset
+
+    def get_ruleset(self, ruleset_name):
+        if ruleset_name not in self.rulesets:
+            raise RulesetNotFoundException(ruleset_name)
+        return self.rulesets[ruleset_name]
+
+    @paginate(pagination_model=PAGINATION_MODEL)
+    def list_rulesets(self):
+        return [self.rulesets[key] for key in self.rulesets] if self.rulesets else []
```

Comment:
```suggestion
        return list(self.rulesets.values())
```
Slightly easier to read, IMO, and it should do the same thing.

Reply:
Ah yes I agree, will change that. Thanks.